generators/app/index.js

```diff
@@ -238,6 +238,9 @@ module.exports = class JHipsterAppGenerator extends BaseBlueprintGenerator {
 
         useBlueprints = !this.fromBlueprint && this.instantiateBlueprints('app');
 
+        if (!this.jhipsterConfig.skipGeneratedFlag) {
+            this.registerGeneratedAnnotationTransform();
+        }
```

Comment:
Move to generator-base too, so it will be registered for every generator.

Reply:
When moving this to generator-base, the `jhipster import-jdl` command does not work anymore (see failing Azure pipelines). Somehow `this.jhipsterConfig.packageName` is `undefined` at this point, but it is needed inside the `registerGeneratedAnnotationTransform()` function.

Reply:
We should not rely that `packageName` is defined that the constructor point.
It will never be defined when creating an app using prompts.

Reply:
Thank you very much @mshima for your extensive help with this PR